frontend/src/pages/scenes/sceneForm/SceneForm.tsx

```diff
@@ -273,6 +278,10 @@ const SceneForm: FC<SceneProps> = ({ scene, initial, callback, saving }) => {
               <Form.Control.Feedback type="invalid">
                 {errors?.date?.message}
               </Form.Control.Feedback>
+              {/* <Form.Text>
+                If the precise date is unknown the day and/or month can be
+                omitted.
+              </Form.Text> */}
```

Comment:
Copied this from `PerformerForm`, but there isn't enough space for this text.
The error message for invalid date is kind of too long now, as well.